doc/source/user_guide/boundary_conditions.rst

```diff
@@ -61,25 +73,51 @@ boundary conditions at inlets.
 
 Copying boundary conditions
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~
-The ``define.boundary_conditions.copy_bc`` TUI command copies boundary conditions
-to other zones.
+The following example shows a comparison between the TUI command and the
+python code for copying boundary conditions to other zones.
+
+TUI command
+
+.. code:: scheme
+
+    /define/boundary-conditions/copy-bc cold-inlet hot-inlet ()
+
+Python command
 
 .. code:: python
 
     session.solver.tui.define.boundary_conditions.copy_bc('cold-inlet','hot-inlet','()')
 
 Listing zones
 ~~~~~~~~~~~~~
-The ``define.boundary_conditions.list_zones`` TUI command prints in the Fluent console
-the types and IDs of all zones.
+The following example shows a comparison between the TUI command and the
+python code for printing the types and IDs of all zones in the Fluent
+console.
```

Comment:
```suggestion
The following example shows a comparison between the TUI command and the
Python code for printing to the Fluent console the types and IDs of all zones.
```

Reply:
Although the edited construction is not conversational in nature, it is grammatically correct. The way it was written implies that the types and IDs of all  zones in the Fluent console are being printed.